doc/source/user_guide/io.rst

```diff
@@ -5080,6 +5091,18 @@ table name and optionally a subset of columns to read.
 
    pd.read_sql_table('data', engine)
 
+.. note::
+
+  Note that pandas infers column dtypes from query outputs, and not by looking
```

Comment:
Does this need to be indented to be under the `n` in note`?

Reply:
It seems not, see for instance the note at line 4884, which is rendered correctly. In general, indentations for the note sections are inconsistent in this file. If there's anything in the guidelines, I missed it (twice now :)),